RecoLocalMuon/GEMRecHit/python/me0RecHits_cfi.py

```diff
@@ -3,5 +3,5 @@
 me0RecHits = cms.EDProducer("ME0RecHitProducer",
     recAlgoConfig = cms.PSet(),
     recAlgo = cms.string('ME0RecHitStandardAlgo'),
-    me0DigiLabel = cms.InputTag("simMuonME0ReDigis"),
+    me0DigiLabel = cms.InputTag("simMuonME0PseudoReDigis"),
```

Comment:
Pseudo means that the digis are of type ME0DigiPreReco. Real digis are of type ME0Digi. The pseudo digitization sequence for ME0 was introduced for performance studies (deciding layers, eta partitions, strips) for the Muon Upgrade TDR. This sequence will be taken out of the main digi step in the future. The real ME0 digi sequence is what will be used to build ME0 triggers and for continued ME0 performance studies.